tests/modules/sightings/resources/utils.py

```diff
@@ -108,3 +110,76 @@ def delete_sighting(flask_app_client, user, sight_guid, expected_status_code=204
         test_utils.validate_dict_response(
             response, expected_status_code, {'status', 'message'}
         )
+
+
+# Create a default valid Sage detection response (to allow for the test to corrupt it accordingly)
+def build_sage_identification_response(job_uuid, annot_uuid, algorithm):
+
+    # Generate the response back from Sage.
+    # 15th July, Do not assume this is correct. This is hacky first try and probably needs
+    # expanding depending on algorithm
```

Comment:
Yep, it will be a bit different but modeling after a Hotspotter identification return takes care of 80% of our plaforms.